src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorIntegrationTest.java

```diff
@@ -406,6 +406,8 @@ public void testVariantRecalibratorSNPscattered(final String[] params) throws IO
         doSNPTest(params, getLargeVQSRTestDataDir() + "/snpTranches.scattered.txt", getLargeVQSRTestDataDir() + "snpRecal.vcf"); //tranches file isn't in the expected/ directory because it's input to GatherTranchesIntegrationTest
     }
 
+    //One of the 8 positive Gaussians (index 4) has weight zero, which is fine, but the one at index 2 has a covariance
```

Comment:
Is there anything in the data or arguments from which we expect a priori that these errors will occur, and at the given indices?

Reply:
In the input data AS_MQrankSum is constant, which I note where the argument string is defined, but I could make that more clear.  Specifying the index of the Gaussian probably isn't informative since anything that touches the random number generator has the potential to change that.

Reply:
In that case a comment here that the constantness (constancy?) of AS_MQrankSum causes one Gaussian to have a degenerate covariance would be helpful.  I would either not even mention that it's the one with index 2 or be explicit that this is arbitrary (likewise for the zero-weight Gaussian at index 4).